### PR TITLE
(selenium-chromium-edge-driver) Build package for latest version of Edge package

### LINKED ISSUE #2241: (selenium-chromium-edge-driver) Outdated
### Checklist

- [X] I have checked the moderation queue that no new version has been submitted (only visible when logged in), or the submitted package version has failed the automated checks.

Please navigate to the following [link](https://community.chocolatey.org/packages?q=&moderatorQueue=true&moderationStatus=all-statuses&prerelease=false&sortOrder=package-download-count) to view the moderation queue.

- [X] I have verified that the new version is a stable release.
- [X] I have looked to see if there are any Bug reports that prevents a new version being submitted and that no existing Outdated Reports have been created.
- [X] I have verified the new version has a Windows binary (EXE or MSI installer, Zip or other archive, etc).

### New Software Version

115.0.1860.0:

### Download location

https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/

### Package Page

https://community.chocolatey.org/packages/selenium-chromium-edge-driver

### PR DESCRIPTION
**Pull request by pascalberger**
## Description

Changes update script to update Selenium driver for Edge to the latest published version of the Edge Chocolatey package.

## Motivation and Context

Currently package was only updated when same version as last version of Edge package was available on the Selenium driver website. This could lead to situations where packages, for versions which are available, but not on homepage, were never published.

Fixes #2241 

## How Has this Been Tested?

Run locally.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [ ] I have updated the package description and it is less than 4000 characters.
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md)
- [ ] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

**Comment by pauby** [comment]
@pascalberger #2267 also looks to address this. Which one is the best to keep?

**Comment by pascalberger** [comment]
> @pascalberger #2267 also looks to address this. Which one is the best to keep?

@pauby Different package: This one is for `selenium-chromium-edge-driver`, https://github.com/chocolatey-community/chocolatey-packages/pull/2267 is for `selenium-chrome-driver`

**Comment by pauby** [comment]
🤦‍♂️ Thanks @pascalberger

File: automatic/selenium-chromium-edge-driver/update.ps1 [hunk 1]
```diff
@@ -38,12 +36,9 @@ function global:au_SearchReplace {
 function global:au_GetLatest {
   # We use the available package on CCR as the source of truth
   $packageVersion = Get-EdgePackageVersion -PackageName 'microsoft-edge'
-  $escapedVersion = [regex]::Escape($packageVersion)
```
Comment (pauby) [inline automatic/selenium-chromium-edge-driver/update.ps1:41]: I'm not entirely sure the reason it's here but I'm assuming it's for a good reason. Is there a reason this was removed?

**Comment by pauby** [comment]
Looking at this again, I actually meant #2247

**Comment by pascalberger** [comment]
Closing since #2247 has been merged which fixes this issue

**Pull request closed without merging**